docs/content/en/blog/articles/release-1.0/index.md

```diff
@@ -99,4 +99,4 @@ And new contributors:
 * [@stanislawwos](https://github.com/stanislawwos)
 * [@tlasica](https://github.com/tlasica)
 
-Looking for a guided experience with K8ssandra? Join us at the [workshop](https://www.datastax.com/workshops/142078180663) next week as we walk through installation and exploration of this cloud-native data platform. Explore the updated [Getting Started](/docs/getting-started/) guide and join the [community](/community) as we move forward towards the next release.
+Looking for a guided experience with K8ssandra? Join us at the [workshop](https://www.datastax.com/workshops/142078180663) next week as we walk through installation and exploration of this cloud native data platform. Explore the updated [Getting Started](/docs/getting-started/) guide and join the [community](/community) as we move forward towards the next release.
```

Comment:
In this case it should be hyphenated. The rule for hyphenation is: if it is a multi-word phrase acting as a single adjective, hyphenate. There's an exception for adverbs ending in "-ly."

```suggestion
Looking for a guided experience with K8ssandra? Join us at the [workshop](https://www.datastax.com/workshops/142078180663) next week as we walk through installation and exploration of this cloud-native data platform. Explore the updated [Getting Started](/docs/getting-started/) guide and join the [community](/community) as we move forward towards the next release.
```